src/main/resources/swagger/api-docs.yaml

```diff
@@ -4115,10 +4115,30 @@ definitions:
       ok:
         type: boolean
         description: whether any system(s) need attention
-
+        default: false
       systems:
         type: object
         description: Map[String, SubsystemStatus]
+        default: {
```

Comment:
Makes sense, changed over.

Further learning: YAML is actually a superset of JSON, with the braces and brackets optionally (and typically) replaced with indentation. YAML's entire purpose in life is to be more human-readable.